src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/ET1ServingControllerTest.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.controllers;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import uk.gov.hmcts.ethos.replacement.docmosis.DocmosisApplication;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.VerifyTokenService;
+
+import java.io.File;
+import java.util.Objects;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(ET1ServingController.class)
+@ContextConfiguration(classes = DocmosisApplication.class)
+public class ET1ServingControllerTest {
+
+    private static final String AUTH_TOKEN = "Bearer eyJhbGJbpjciOiJIUzI1NiJ9";
+    private static final String SERVING_DOCUMENT_OTHER_TYPE_NAMES_URL = "/midServingDocumentOtherTypeNames";
+    private static final String SERVING_DOCUMENT_RECIPIENT_URL = "/midServingDocumentRecipient";
+
+    @Autowired
+    private WebApplicationContext applicationContext;
+
+    @MockBean
+    private VerifyTokenService verifyTokenService;
+
+    private MockMvc mvc;
+    private JsonNode requestContent;
+
+    @Before
+    public void setUp() throws Exception {
+        mvc = MockMvcBuilders.webAppContextSetup(applicationContext).build();
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        requestContent = objectMapper.readTree(new File(Objects.requireNonNull(getClass()
+                .getResource("/servingRequest.json")).toURI()));
+    }

```

Comment:
mvc doesn't need going in setup method does it?  That could be inline?

Would be better done as @BeforeEach for requestContext to reset for each test